Class-Code/Game of Life/jkohl/sketch.js

```diff
@@ -1,14 +1,77 @@
-//Your Name(s)
-//Project Name
-//Date
+//Jacob Kohl
+//Game of Life
+//March 15, 2019
 
-/* A brief description of what the program is */
 
+let current = [];
+let cellsize = 10;
+
+let columns,rows;
 
 function setup() {
-    createCanvas(400, 400);
+    createCanvas(900, 500);
+
+    columns = floor(width/cellsize);
+    rows = floor(height/cellsize)
+
+    for(let row = 0; row < rows; row++){
+        current.push(new Array(columns).fill(0).map(()=>round(random())))
+    }
+    noStroke();
+    current[5][5] = 1
+    current[5][6] = 1
+    current[5][7] = 1
+    //noLoop()
```

Comment:
A comment here would be good. What's happening here if you are already initializing the board with random 0's and 1's